<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>932</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>925</width>
    <height>932</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>925</width>
    <height>932</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>60</width>
      <height>841</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color :#dedcff;
border:0;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background:white;
	border-radius: 20px;
	transition:0,5 s;
}
QPushButton:hover {
	background:black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background:white;
	border-radius: 20px;
	transition:0,5 s;
}
QPushButton:hover {
	background:black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background:white;
	border-radius: 20px;
	transition:0,5 s;
}
QPushButton:hover {
	background:black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>871</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>60</number>
     </property>
     <property name="rightMargin">
      <number>70</number>
     </property>
     <item>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">background-color: #acabb0;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(172, 171, 176, 255), stop:0.74 rgba(224, 28, 52, 255));
border:0px;
border-radius:12px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="input">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>400</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white;
border-radius:20px;
padding-left: 50px;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="rechercher">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>10</y>
          <width>40</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background:white;
	border-radius: 20px;
	transition:0,5 s;
}
QPushButton:hover {
	background:black;
}</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
       </widget>
       <widget class="QPushButton" name="listeL1">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>51</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>L1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="listeL2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>90</y>
          <width>51</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>L2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>90</y>
          <width>51</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>L3</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>90</y>
          <width>51</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>M1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>90</y>
          <width>51</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>M2</string>
        </property>
       </widget>
       <widget class="QComboBox" name="mode">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>10</y>
          <width>91</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;
border-radius:20px;
border:0;</string>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Num</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MacE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MacW</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Serial</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Etat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ip</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tout</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>90</y>
          <width>51</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>All</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>51</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>Par:</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>821</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>807</width>
       <height>337</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>861</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="styleSheet">
        <string notr="true">background-color :#dedcff;
border:0;
border-radius:12px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>191</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre total des étudiants</string>
        </property>
       </widget>
       <widget class="QLabel" name="tEt">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>67</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_4">
       <property name="styleSheet">
        <string notr="true">background-color :#dedcff;
border:0;
border-radius:12px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>191</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre total des panasonics</string>
        </property>
       </widget>
       <widget class="QLabel" name="tPan">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>67</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_6">
       <property name="styleSheet">
        <string notr="true">background-color :#dedcff;
border:0;
border-radius:12px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre total des occurences</string>
        </property>
       </widget>
       <widget class="QLabel" name="tOc">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>67</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
